addons/web/static/src/views/fields/statusbar/statusbar_field.js

```diff
@@ -144,8 +144,13 @@ export class StatusBarField extends Component {
                 {
                     category: "smart_action",
                     hotkey: "alt+x",
-                    isAvailable: () =>
-                        !this.props.isDisabled && !this.getAllItems().at(-1).isSelected,
+                    isAvailable: () => {
+                        return (
+                            !this.props.isDisabled &&
+                            this.getAllItems().length &&
+                            !this.getAllItems().at(-1).isSelected
+                        );
```

Comment:
```suggestion
                        return (
                            !this.props.isDisabled &&
                            !this.getAllItems().at(-1)?.isSelected
                        );
```
to avoid going through getAllItems twice

Reply:
```suggestion
                        if (this.props.isDisabled) {
                            return false;
                        }
                        const items = this.getAllItems();
                        return items.length && !items.at(-1).isSelected;
```
then